src/navigations/root-stack.js

```diff
@@ -60,14 +60,17 @@ import {InitialStartupAtom, LoadingStartupContext} from '../service/initialStart
 import {NavigationConstants} from '../utils/constants';
 import {followersOrFollowingAtom} from '../screens/ChannelListScreen/model/followersOrFollowingAtom';
 import {useInitialStartup} from '../hooks/useInitialStartup';
+import {resendPendingMessages, useSendAnonMessage, useSendSignedMessage} from '../service/chat';
 
 const RootStack = createNativeStackNavigator();
 
 export const RootNavigator = () => {
   const initialStartup = useRecoilValue(InitialStartupAtom);
   const [following, setFollowing] = useRecoilState(followersOrFollowingAtom);
   const loadingStartup = useInitialStartup();
-  useLocalDatabaseHook(true);
+  const {localDb} = useLocalDatabaseHook(true);
+  const sendChatSignedMutation = useSendSignedMessage();
+  const sendChatAnonMutation = useSendAnonMessage();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new imports and mutation functions are well defined. However, it's important to handle potential errors that might occur during the execution of these mutations. You should consider wrapping `useSendSignedMessage` and `useSendAnonMessage` in a try-catch block or provide an error handling mechanism within these functions.

```diff
- const sendChatSignedMutation = useSendSignedMessage();
- const sendChatAnonMutation = useSendAnonMessage();

+ let sendChatSignedMutation;
+ let sendChatAnonMutation;
+ try {
+   sendChatSignedMutation = useSendSignedMessage();
+   sendChatAnonMutation = useSendAnonMessage();
+ } catch (error) {
+   console.error("Error initializing chat mutations: ", error);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->